RadixWallet/Profile/ImportLegacyWalletModels/OlympiaAccountToMigrate.swift

```diff
@@ -37,7 +37,7 @@ public struct OlympiaAccountToMigrate: Sendable, Hashable, CustomDebugStringConv
 	public var debugDescription: String {
 		"""
 		accountType: \(accountType)
-		name: \(displayName ?? "")
+		name: \(displayName ?? "Unknown")
```

Comment:
This shocked me until I saw that it was a debug description

Reply:
this actually prevented a crash, since type is `NonEmptyString`, which apparently conforms to `ExpressibleByStringLiteral` => crash.